docs/Guides-RootContainer.md

```diff
@@ -142,4 +142,4 @@ When `forceFetch` is true and `renderFetched` is called as a result of available
 
 **Relay.RootContainer** also supports the `onReadyStateChange` prop which lets us receive fine-grained events as they occur while fulfilling the data requirements.
 
-Learn how to use this prop in the [Ready State guide](guides-ready-state.html).
+Learn how to use `onReadyStateChange` in our next guide, Ready State.
```

Comment:
I omitted it because there is a link ~20 pixels down that takes you to the Next page, but I can add it back in!